inference-engine/src/cldnn_engine/ops/scatter_nd_update.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
nit: please update year in copyright here and in the other files.

Reply:
I updated all files to 2021